runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -4014,14 +4014,18 @@ class INTERPRETER_CLASS
 					true);
 
 				if (NULL == result) {
-					result = VM_ValueTypeHelpers::getFlattenableFieldAtOffset(
+					buildGenericSpecialStackFrame(REGISTER_ARGS, 0);
```

Comment:
Please add a comment about this change into your commit message and the [description](https://github.com/eclipse-openj9/openj9/pull/14564#issue-1143417253) of this PR.

Reply:
Approve assuming the above comment is addressed.